staff/nicolas-martin/cv/index.html

```diff
@@ -0,0 +1,182 @@
+                    Tempor por la gloria de mi madre aliqua qué dise usteer. Mamaar minim caballo blanco caballo negroorl dolor condemor nostrud te voy a borrar el cerito et ad ese que llega minim. Et irure veniam ex voluptate consequat.
+                </p>
+            </section>
+
+            <section>
+                <header>
+                    <h3>Mars Company - <time datetime="2008-02-14">01-01-2009</time></h3>
+                </header>
+            
+                <p>Lorem fistrum ese hombree ad al ataquerl. Tiene musho peligro ut eiusmod hasta luego Lucas enim. Et condemor diodeno amatomaa labore. Ahorarr reprehenderit esse aliquip amatomaa consequat está la cosa muy malar dolor. Hasta luego Lucas ex te va a hasé pupitaa pupita papaar papaar. 
+                    Tempor por la gloria de mi madre aliqua qué dise usteer. Mamaar minim caballo blanco caballo negroorl dolor condemor nostrud te voy a borrar el cerito et ad ese que llega minim. Et irure veniam ex voluptate consequat.
+                </p>
+            </section>
+
+            <section>
+                <header>
+                    <h3>Mars Company - <time datetime="2008-02-14">01-01-2009</time></h3>
+                </header>
+            
+                <p>Lorem fistrum ese hombree ad al ataquerl. Tiene musho peligro ut eiusmod hasta luego Lucas enim. Et condemor diodeno amatomaa labore. Ahorarr reprehenderit esse aliquip amatomaa consequat está la cosa muy malar dolor. Hasta luego Lucas ex te va a hasé pupitaa pupita papaar papaar. 
+                    Tempor por la gloria de mi madre aliqua qué dise usteer. Mamaar minim caballo blanco caballo negroorl dolor condemor nostrud te voy a borrar el cerito et ad ese que llega minim. Et irure veniam ex voluptate consequat.
+                </p>
+            </section>
+
+            <section>
+                <header>
+                    <h3>Mars Company - <time datetime="2008-02-14">01-01-2009</time></h3>
+                </header>
+            
+                <p>Lorem fistrum ese hombree ad al ataquerl. Tiene musho peligro ut eiusmod hasta luego Lucas enim. Et condemor diodeno amatomaa labore. Ahorarr reprehenderit esse aliquip amatomaa consequat está la cosa muy malar dolor. Hasta luego Lucas ex te va a hasé pupitaa pupita papaar papaar. 
+                    Tempor por la gloria de mi madre aliqua qué dise usteer. Mamaar minim caballo blanco caballo negroorl dolor condemor nostrud te voy a borrar el cerito et ad ese que llega minim. Et irure veniam ex voluptate consequat.
+                </p>
+            </section>
+            
+        </section>     
+
+        <section id="for">
+            <header>
+                <h1>Formación</h1>
+                <p>Lorem fistrum torpedo occaecat a gramenawer velit te va a hasé pupitaa ad.</p>
+            </header>
+
+            <ul>
+                    <li><h4>Universidad estelar</h4>
+                    From: <time datetime="2008-02-14">01-01-2009</time>
```

Comment:
Tab first <li>

Reply:
sorry but i don't understand 😞

Reply:
I've applied the shortcut in Visual Studio Editor